src/agent/GRPCClientAgent.ts

```diff
@@ -1,7 +1,7 @@
 import type { TLSConfig } from '../network/types';
 
 import { GRPCClient, utils as grpcUtils } from '../grpc';
-import * as agentPB from '../proto/js/Agent_pb';
+import { messages } from '.';
```

Comment:
Don't import from the current directory's index. We only do that from outside the directory (and even then, it's usually intended for exports). In our own code, we should be always as specific as possible. And furthermore we should be using `agentPB`. Because `messages` is too vague.

Reply:
Fixing this now.